internal/service/ingestion_task_service_test.go

```
@@ -534,33 +399,33 @@ func TestIngestionTaskServiceCreateAndEnqueueRejectsActiveExistingTask(t *testin
 
-func TestIngestionTaskServiceCreateAndEnqueueRollsBackNewTaskOnPublishFailure(t *testing.T) {
+func TestIngestionTaskServiceCreateAndEnqueueKeepsNewTaskScheduledOnPublishFailure(t *testing.T) {
 	db := setupServiceTestDB(t)
 	pushServiceDB(t, db)
 	publisher := &recordingTaskPublisher{err: errors.New("publish failed")}
 	svc := NewIngestionTaskService()
 	svc.taskPublisher = publisher
 
 	ctx := t.Context()
-	_, err := svc.CreateAndEnqueue(ctx, &entity.IngestionTask{
+	task, err := svc.CreateAndEnqueue(ctx, &entity.IngestionTask{
```

golangci-lint (2.12.2)

[error] SA4006: this value of task is never used

(staticcheck)